pyuvdata/utils.py

```diff
@@ -1156,7 +1156,7 @@ def get_baseline_redundancies(baselines, baseline_vecs, tol=1.0, with_conjugates
 
     lens = np.sqrt(np.sum(vec_bin_centers**2, axis=1))
 
-    return bl_gps, vec_bin_centers, lens
+    return bl_gps, vec_bin_centers, lens, None
```

Comment:
I'm a bit confused by returning `None` here. I'm not sure it's a better interface to ensure that we return the same number of things by returning a `None`. In fact, this recently led to confusion in the beam interp functions (see issue #622). @jpober what do you think?